src/test/kotlin/no/nav/familie/ba/sak/kjerne/dokument/domene/TestKlasser.kt

```diff
@@ -2,49 +2,117 @@ package no.nav.familie.ba.sak.kjerne.dokument.domene
 
 import com.fasterxml.jackson.annotation.JsonSubTypes
 import com.fasterxml.jackson.annotation.JsonTypeInfo
+import no.nav.familie.ba.sak.common.MånedPeriode
 import no.nav.familie.ba.sak.common.Utils
-import no.nav.familie.ba.sak.ekstern.restDomene.RestPerson
+import no.nav.familie.ba.sak.kjerne.autovedtak.fødselshendelse.Resultat
 import no.nav.familie.ba.sak.kjerne.behandlingsresultat.UregistrertBarnEnkel
 import no.nav.familie.ba.sak.kjerne.beregning.domene.YtelseType
+import no.nav.familie.ba.sak.kjerne.dokument.UtvidetScenario
+import no.nav.familie.ba.sak.kjerne.endretutbetaling.domene.Årsak
 import no.nav.familie.ba.sak.kjerne.grunnlag.personopplysninger.Målform
+import no.nav.familie.ba.sak.kjerne.grunnlag.personopplysninger.PersonType
+import no.nav.familie.ba.sak.kjerne.vedtak.begrunnelser.VedtakBegrunnelseSpesifikasjon
+import no.nav.familie.ba.sak.kjerne.vedtak.begrunnelser.VedtakBegrunnelseType
+import no.nav.familie.ba.sak.kjerne.vedtak.begrunnelser.tilSanityBegrunnelse
 import no.nav.familie.ba.sak.kjerne.vedtak.domene.BegrunnelseData
-import no.nav.familie.ba.sak.kjerne.vedtak.domene.BegrunnelsePerson
-import no.nav.familie.ba.sak.kjerne.vedtak.vedtaksperiode.RestVedtaksbegrunnelse
-import no.nav.familie.ba.sak.kjerne.vedtak.vedtaksperiode.UtbetalingsperiodeDetalj
+import no.nav.familie.ba.sak.kjerne.vedtak.domene.MinimertPerson
 import no.nav.familie.ba.sak.kjerne.vedtak.vedtaksperiode.Vedtaksperiodetype
+import no.nav.familie.ba.sak.kjerne.vedtak.vedtaksperiode.domene.BrevBegrunnelseGrunnlag
+import no.nav.familie.ba.sak.kjerne.vedtak.vedtaksperiode.domene.MinimertEndretUtbetalingAndel
+import no.nav.familie.ba.sak.kjerne.vedtak.vedtaksperiode.domene.MinimertPersonResultat
+import no.nav.familie.ba.sak.kjerne.vedtak.vedtaksperiode.domene.MinimertUtbetalingsperiodeDetalj
+import no.nav.familie.ba.sak.kjerne.vedtak.vedtaksperiode.domene.MinimertVilkårResultat
+import no.nav.familie.ba.sak.kjerne.vilkårsvurdering.domene.AnnenVurdering
+import no.nav.familie.ba.sak.kjerne.vilkårsvurdering.domene.Vilkår
 import java.math.BigDecimal
 import java.time.LocalDate
 
-data class UtbetalingsperiodeDetaljEnkel(
+data class BrevBegrunnelseTestConfig(
+    val beskrivelse: String,
+
+    val fom: LocalDate,
+    val tom: LocalDate?,
+    val vedtaksperiodetype: Vedtaksperiodetype,
+    val begrunnelser: List<BrevBegrunnelseGrunnlagConfig>,
+    val fritekster: List<String>,
+
+    val personerPåBehandling: List<PersonPåBehandlingConfig>,
+
+    val utvidetScenario: UtvidetScenario = UtvidetScenario.IKKE_UTVIDET_YTELSE,
+    val uregistrerteBarn: List<UregistrertBarnEnkel>,
+    val erFørsteVedtaksperiodePåFagsak: Boolean = false,
+    val målformSøker: Målform,
```

Comment:
```suggestion
    val brevMålform: Målform,
```

Det viktige her er vel hvilken målform brevet skal ha. Akkurat nå bestemmes det av søkers målform, men det blir fort en sterk avhengighet og kan kanskje bedre kobles fra hverandre om man snakker om det mer generelt.

Reply:
Yes, må nok endres flere steder